spec/predicates/cyclonedx.md

```diff
@@ -2,7 +2,7 @@
 
 Type URI: https://cyclonedx.org/bom
```

Comment:
This should be updated to match the `predicateType` below, no? Also, may be good to add in the predicate name here, per https://github.com/in-toto/ITE/pull/52.